scripts/lc-builds/blueos_nvcc10_caliper25_gcc8.3.1.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+# and RAJA Performance Suite project contributors.
+# See the RAJAPerf/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#################################################################################
+
+BUILD_SUFFIX=lc_blueos-nvcc10-caliper-gcc8.3.1
+RAJA_HOSTCONFIG=../tpl/RAJA/host-configs/lc-builds/blueos/nvcc_gcc_X.cmake
+
+rm -rf build_${BUILD_SUFFIX} >/dev/null
+mkdir build_${BUILD_SUFFIX} && cd build_${BUILD_SUFFIX}
+
+module load cmake/3.14.5
+module load caliper-2.5.0-gcc-8.3.1-cu3vy3k
+
+CALIPER_PREFIX=/usr/WS2/holger/spack/opt/spack/linux-rhel7-power9le/gcc-8.3.1/caliper-2.5.0-cu3vy3kjwjerpdm6xis2kauhz4s6wto2/
```

Comment:
Yes, I can get started on the Spack bit. What are your thoughts on the public location?

Reply:
We have space in /usr/gapps/raja-dev on both the RZ and the CZ. So that's probably where we want this stuff. What I'm not clear on is what we need to install and how much maintenance will it require. My goal is to be able to curate and monitor caliper data from perf suite runs via Gitlab CI testing. Do we need a build and installation of caliper and adiak for each compiler and version we test? How often should we be updating them? Things like that....

Reply:
The Caliper services that we mainly use are independent of arch on the cpu side (so as long as you can perform the link you can use and advance different compilers without issue. But, when you want to use architecture specific features mpi, roctx, cuda-activity-report, etc then yeah, best to keep version specific installs. I know that the RAJA team is deep in this question.

Reply:
The minimum required is a bare-bones CPU-specific build of Caliper (x86, P9).  Additional Caliper services (PAPI on the CPUs, Nvidia or AMD GPU services) require architecture specific installs.  Mike ends up with 3 flavors: CPU-only, P9+cuda, AMD